src/app/components/zlux-file-explorer/zlux-file-explorer.component.ts

```diff
@@ -96,6 +101,54 @@ export class ZluxFileExplorerComponent implements OnInit, OnDestroy {
     // }
     // this.persistentDataService.setData(obj)
     //   .subscribe((res: any) => { });
+    switch(this.theme) { 
+      case 'carbon': { 
+         this.headerStyle =  {
+          'background-color': '#3d70b2',
+          'color': 'white',
+          'width':'99.7%',
+          'text-align':'left',
+          'padding-left': '7px'
+        };
+         this.inputStyle = {
+          'background-color': '#eee', 
+          'color': 'black', 
+          'border':'2px solid #3d70b2',
+          'margin-top': '20px'
+        };
+        this.searchStyle = {
+          'min-width': '250px',
+          'display': 'inline-block',
+          'height': '40px',
+          'width':'90%',
+    
+        };
+  
+        this.treeStyle = {
+          'color':'#646464'
+        };
+  
+        this.style = {
+          'background-color':'#F4F7FB',
+          'margin-top': '10px',
+          'max-height':'320px',
+          'overflow-y': 'scroll'
+          
+          
+        };
+  
+         break; 
+      } 
+  
+      default: { 
+        // this.headerStyle = {'background-color': '#464646'};
+        this.treeStyle = {'filter': 'brightness(3)'};
+        // this.style = {'background-color': '#464646'};
```

Comment:
Can you hardcode a background style here for "default". By hardcoding the style, you fix the Workflow bug where Workflow bleeds into FE. You fixed most of it via the ! but as you can see, because the background color isn't specified, it still breaks

![image](https://user-images.githubusercontent.com/20528015/62251421-b0540d00-b3bd-11e9-93d9-2d0a84d35f23.png)